wordpress-mobile/WordPress-iOS
# Create Installable Builds with Buildkite

Moves installable builds from CircleCI to Buildkite. 

Uses the release toolkit to post the installable build instructions instead of writing out a file for Peril – apart from that, it works the same way.

To test:
- Note that all the tasks on this PR pass.
- Download the IPA files from the links.
- Verify that the builds show up in App Center.
- Extra ~paranoia~ credit: try installing the App Center builds to ensure they work.

## Regression Notes
1. Potential unintended areas of impact
Could theoretically break installable builds.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Checked that they were actually posted to App Center.

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
